compiler/ir/ir.tree/tree-generator/src/org/jetbrains/kotlin/ir/generator/IrTree.kt

```diff
@@ -183,12 +183,14 @@ object IrTree : AbstractTreeBuilder() {
     }
     val declarationContainer by element(Declaration) {
         parent(declarationParent)
+        ownsChildren = false
```

Comment:
Minor: This line (and others in this commit) does not adhere to the style of the dsl, which is:

```
Metadata config

Parents

Fields
```

(look at other elements). I think this is important for readability.
Maybe comment it at the top?